docs/UserGuide.md

```diff
@@ -4,38 +4,25 @@
   pageNav: 3
 ---
 
-# Y2Direct User Guide
+# PawFection User Guide
 
-Y2Direct (Y2D) is a **desktop app for managing contacts, optimized for use via a Line Interface** (CLI) while still
-having the benefits of a Graphical User Interface (GUI). It aims to provide a platform for students to store
-work/school-related contacts. It will elevate user experience through organized contact categorization and task
-tagging.
+PawFection is a **desktop app for managing animals in a pet shelter, optimized for use via a Line Interface** (CLI) while still
```

Comment:
There may be a type here! Did you meant "Command Line Interface (CLI)" rather than "Line Interface (CLI)"

Reply:
Yes I agree, apologies for the typo!